lovefield/lovefield.d.ts

```diff
@@ -188,7 +188,7 @@ declare namespace lf {
       local: string
       ref: string
       action: lf.ConstraintAction
-      timing: lf.ConstraintAction
+      timing: lf.ConstraintTiming
```

Comment:
Can you modify this to be identical to https://github.com/google/lovefield/blob/master/dist/lovefield.d.ts#L212-216?